kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -535,6 +539,13 @@ def parse(s: str) -> List[T]:
         help='Store every Nth state in the CFG for inspection.',
     )
 
+    explore_args.add_argument(
+        '--with-custom-rpc',
+        dest='rpc_command',
+        type=str,
+        help='Custom command to start RPC server',
+    )
```

Comment:
```suggestion
    explore_args.add_argument(
        '--with-custom-rpc',
        dest='rpc_command',
        type=str,
        default='kore-rpc',
        help='Custom command to start RPC server',
    )
```

Similarly, we _could_ set the default here. Does it make sense to do so?

Reply:
According to Tamas' preference, it is better to use an optional parameter